superset/connectors/sqla/models.py

```diff
@@ -930,39 +930,48 @@ def _get_sqla_row_level_filters(
 
     def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-many-branches,too-many-statements
         self,
-        metrics: Optional[List[Metric]] = None,
-        granularity: Optional[str] = None,
-        from_dttm: Optional[datetime] = None,
-        to_dttm: Optional[datetime] = None,
+        apply_fetch_values_predicate: bool = False,
         columns: Optional[List[str]] = None,
-        groupby: Optional[List[str]] = None,
+        extras: Optional[Dict[str, Any]] = None,
         filter: Optional[  # pylint: disable=redefined-builtin
             List[QueryObjectFilterClause]
         ] = None,
-        is_timeseries: bool = True,
-        timeseries_limit: int = 15,
-        timeseries_limit_metric: Optional[Metric] = None,
-        row_limit: Optional[int] = None,
-        row_offset: Optional[int] = None,
+        from_dttm: Optional[datetime] = None,
+        granularity: Optional[str] = None,
+        groupby: Optional[List[str]] = None,
         inner_from_dttm: Optional[datetime] = None,
         inner_to_dttm: Optional[datetime] = None,
+        is_rowcount: bool = False,
+        is_timeseries: bool = True,
+        metrics: Optional[List[Metric]] = None,
         orderby: Optional[List[OrderBy]] = None,
-        extras: Optional[Dict[str, Any]] = None,
         order_desc: bool = True,
-        is_rowcount: bool = False,
-        apply_fetch_values_predicate: bool = False,
+        to_dttm: Optional[datetime] = None,
+        series_columns: Optional[List[str]] = None,
+        series_limit: Optional[int] = None,
+        series_limit_metric: Optional[Metric] = None,
+        row_limit: Optional[int] = None,
+        row_offset: Optional[int] = None,
+        timeseries_limit: int = 15,
+        timeseries_limit_metric: Optional[Metric] = None,
     ) -> SqlaQuery:
         """Querying any sqla table from this common interface"""
         template_kwargs = {
+            "columns": columns,
             "from_dttm": from_dttm.isoformat() if from_dttm else None,
             "groupby": groupby,
             "metrics": metrics,
             "row_limit": row_limit,
             "row_offset": row_offset,
             "to_dttm": to_dttm.isoformat() if to_dttm else None,
+            "table_columns": [col.column_name for col in self.columns],
             "filter": filter,
-            "columns": [col.column_name for col in self.columns],
```

Comment:
Here I'm renaming `columns` to `table_columns`, as `columns` should refer to the actual query object property (I doubt the old `columns` property is really being actively used).